source/client/output_formatter_impl.cc

```diff
@@ -184,8 +185,11 @@ const nighthawk::client::Statistic& FortioOutputFormatterImpl::getRequestRespons
 
 std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) const {
   nighthawk::client::FortioResult fortio_output;
-
-  fortio_output.set_labels("Nighthawk");
+  std::string labels;
+  for (const auto& label : output.options().labels()) {
+    labels += label + " ";
```

Comment:
This feels like we are taking multiple labels and concatenating them, into one label. Is that the behavior? if yes, can we document it next to the new labels option?

Alternatively, maybe the FortioResult message expects a space separated list of labels? If that is so, can we document it here?

https://github.com/envoyproxy/nighthawk/blob/68e59719fe754401907c16e53156bf18cd0e4e65/api/client/transform/fortio.proto#L18

Reply:
Added a descriptive comment in the proto: 3cb9f42354387ed6bca47bb241c87b64087a0dc7